clang/test/CIR/IR/module.cir

```diff
@@ -0,0 +1,9 @@
+// RUN: cir-tool %s -verify-diagnostics -split-input-file
```

Comment:
Nice! This should also roundtrip, you need to also check for the CIR output in order to check printing works too.